source/js/components/currency-selector.js

```diff
@@ -0,0 +1,115 @@
+class CurrencySelect {
+  static selector() {
+    return "#id_currency-switcher-currency";
+  }
+
+  constructor(node) {
+    this.selectMenu = node;
+    this.formContainer = document.getElementById("js-donate-form");
+    this.data = JSON.parse(document.getElementById("currencies").innerHTML);
+    this.oneOffContainer = document.getElementById("js-donate-form-single");
+    this.monthlyContainer = document.getElementById("js-donate-form-monthly");
+    this.defaultCurrency = document.getElementById(
```

Comment:
@nicklee I reckon that just `document.getElementById("id_currency-switcher-currency").value` does the trick here.

Reply:
Updated